corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -13,10 +13,18 @@
         }
     </style>
 {% endblock %}
+{% block js %}{{ block.super }}
+    <script src="{% new_static 'reports/javascripts/reports.config.js' %}"></script>
+    <script src="{% new_static 'reports/javascripts/reports.util.js' %}"></script>
+    <script src="{% new_static 'reports/javascripts/reports.async.js' %}"></script>
+{% endblock %}
```

Comment:
didn't look closely at this, but is there any chance it's easy to have this extend one of the core reporting base templates? seems like we could find ourselves in a situation where it forks a lot of the same code if it's reimplementing a lot of that stuff (this is 100% my fault for starting it outside the framework, but something I was hoping might change with this work)